src/components/Individual_Article/Individual_Article/Article.jsx

```diff
@@ -15,14 +17,26 @@ const Article = ({articleId}) => {
     }
     if (isLoading) return <p>Loading...</p>
 
+    const handleVote = (vote) => {
+        patchVote(articleId, vote).catch(()=> {
+            setVoteChange((currVoteChange) => {return currVoteChange - vote})
+            setErr('Something went wrong, please try again.');
+        })
+        setVoteChange((currVoteChange) => {return currVoteChange + vote})
+        setErr(null);
+    }
```

Comment:
I like that you've considered error handling for this and that you're changing the votes displayed to the user _optimistically_ on line 25! 👍 

To help readability of your code you could move lines 25 and 26 to be before your `patchVote` because of async those lines are executing before the `patchVote` resolves or rejects, so seeing them come before allows the code to be read top to bottom